uber/config.py

```diff
@@ -120,6 +120,18 @@ def PAGE_PATH(self):
     def PAGE(self):
         return cherrypy.request.path_info.split('/')[-1]
 
+    @property
+    def SUPPORTER_COUNT(self):
+        with sa.Session() as session:
+            attendees = session.query(sa.Attendee)
+            individual_supporters = attendees.filter(or_(sa.Attendee.paid == self.HAS_PAID,
+                                                         sa.Attendee.paid == self.REFUNDED),
+                                                         sa.Attendee.amount_extra == c.SUPPORTER_LEVEL).count()
+            group_supporters = attendees.join(sa.Attendee.group).filter(sa.Attendee.paid == self.PAID_BY_GROUP,
+                                                                        sa.Attendee.amount_extra == c.SUPPORTER_LEVEL,
+                                                                        sa.Group.amount_paid > 0).count()
```

Comment:
That's a good point. Though it technically doesn't need to be fixed for launching magstock, it's better to have it work properly for all cases.